books/free-programming-books-langs.md

```diff
@@ -2205,6 +2205,7 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### Ruby
 
+* [A Brief Introduction to Ruby](https://markm208.github.io/rubybook) - Mark Mahoney
```

Comment:
this should go in the interactive tutorials list

Reply:
I am making the change as requested.